src/components/PasswordInput/PasswordInput.tsx

```diff
@@ -0,0 +1,71 @@
+import styled from 'styled-components';
+import { useState } from 'react';
+
+import Text from '../Text';
+import { TextButton } from '../Buttons';
+import TextInput from '../TextInput';
+import Tooltip from '../Tooltip';
+
+type Props = {
+  label: string;
+  onChange: (value: string) => void;
+  showPasswordText: string;
+  tooltip?: string;
+  value: string;
+};
+
+export const PasswordInput = ({
+  label,
+  onChange,
+  showPasswordText,
+  tooltip,
+  value,
+}: Props): JSX.Element => {
+  const inputId = Math.random().toString();
```

Comment:
I would probably bring [uid](https://www.npmjs.com/package/uid)-library for this.
Then would create a utility-function for creating unique id's

Reply:
Yeah, this was just copy-pasted from the TextInput component, but that would be a great refactor!

Reply:
Done!